src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -55,13 +55,14 @@ private function copyFiles(array $manifest, string $from, string $to)
     private function removeFiles(array $manifest, string $from, string $to)
     {
         foreach ($manifest as $source => $target) {
-            $targetPath = $this->concatenatePathParts([$to, $target]);
+            $targetPath = $this->filePathUtil->concatenatePathParts([$to, $target]);
             if ('/' === substr($source, -1)) {
-                $this->removeFilesFromDir($this->concatenatePathParts([$from, $source]), $this->concatenatePathParts([$to, $target]));
+                $this->removeFilesFromDir($this->filePathUtil->concatenatePathParts([$from, $source]), $this->filePathUtil->concatenatePathParts([$to, $target]));
             } else {
                 @unlink($targetPath);
-                $this->write(sprintf('Removed file <fg=green>"%s"</>', $this->relativizePath($targetPath)));
+                $this->write(sprintf('Removed <fg=green>"%s"</>', $this->filePathUtil->relativizePath($targetPath)));
             }
+            $this->removeDirectoryIfEmpty(dirname($targetPath));
```

Comment:
I have added this because as far as I can tell, you can have empty directories left over. I can remove this if need be.